app/jobs/autoscaling.py

```diff
@@ -73,21 +65,9 @@ def get_asg_list():
     asg_list = json.loads(asg_json).get('AutoScalingGroups', [])
     return asg_list
 
-def required_capacity(num_jobs):
-    if num_jobs == 0:
-        result = exactly(0)
-    elif num_jobs == 1:
-        # Often a test job or automatic benchmark job
-        result = exactly(4)
-    elif 1 < num_jobs <= 6:
-        result = exactly(8)
-    else:
-        # This means, as long as we're in this regime, a DesiredCapacity > 24 set manually by the operator
-        # will be respected and not scaled down until we exit this regime.
-        result = at_least(24)
-    return result
-
-def autoscaling_update(my_num_jobs, my_environment,
+def autoscaling_update(num_gsnap_chunks, num_rapsearch_chunks,
```

Comment:
Have you considered passing autoscaling_config through to the ASG constructor?   Then you wouldn't have to pass a list of parameters derived from it.   Long parameter lists are error prone.

Reply:
ok done